gammapy/spectrum/spectral_grouping.py

```diff
@@ -0,0 +1,133 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+from astropy.coordinates import Angle
+from astropy.units import Quantity
+import numpy as np
+from ..spectrum.spectrum_extraction import SpectrumObservationList, SpectrumObservation
+from ..data import ObservationGroupAxis, ObservationGroups
+
+__all__ = [
```

Comment:
@joleroi 
I added this line in the init:
from .spectral_grouping import *

But when I run the python setup.py build_sphinx, I got that error:
```
  File "/Users/jouvin/gammapy/build/lib.macosx-10.9-x86_64-2.7/gammapy/spectrum/SpectralGrouping.py", line 25
    +                               ZenRange=[0., 70.], NZenbin=30):
    ^
SyntaxError: invalid syntax
Sphinx Documentation subprocess failed with return code 1

```